### DISCUSSION TITLE
MacOS - "Cannot connect to the docker socket. Cannot proceed."

### DISCUSSION THREAD

**Discussion by Anendil** [nextcloud/all-in-one#2444] [Questions] [answered]
# Steps to reproduce

   - Install Docker desktop for Apple chip from offical website
   - Run docker command ajusted for mac usage :

```
# For Linux and without a web server or reverse proxy (like Apache, Nginx and else) already in place:
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock.raw:/var/run/docker.sock:ro \
-e DOCKER_SOCKET_PATH=/var/run/docker.sock \
nextcloud/all-in-one:latest
```

- "Cannot connect to the docker socket. Cannot proceed." message loop after restart

# Expected behavior

I expect to have correct creation of the container

# Actual behavior

The container always restart and show "Cannot connect to the docker socket. Cannot proceed."
# Host OS

MacOS 13.3.1 (22E261)
# Nextcloud AIO version

latest
# Current channel

cannot access to AIO interface
# Other valuable info

I tried many potential solutions but nothing work has expected. Tried different installation of docker and with and without special ajustment for macOS. I also tried with docker compose.

Please ask me if more information is needed and what exactly because i am still a beginner with docker :)

**Comment by szaimen** [comment, marked as answer]
Hi, I have the assuption that the Docker Desktop update v4.19 broke things. See https://github.com/nextcloud/all-in-one/issues/2450
